tox.ini

```diff
@@ -44,25 +45,23 @@ skip_missing_interpreters = true
 skipsdist = false
 
 [testenv]
-passenv = CONFIG_SUPPORT_TOKEN_ENV
-whitelist_externals = echo
-deps = echo
+passenv = CONFIG_SUPPORT_TOKEN_ENV,GRAPHVIZ_DOT_BINARY
```

Comment:
Not sure why this was working in Azure before without the GRAPHVIZ_DOT_BINARY env variable, but locally it would not work until I added this (even after I installed graphviz and set the var).

Reply:
Interesting, which test was failing locally without that?

Reply:
Failed test: testMakeSelectedCategoryFigures (testNeighborFigures.NeighborFiguresTests)

```
  File "/Users/aliciaevans/dev/py-mmcif_sitegen/mmcif/sitegen/dictionary/NeighborFigures.py", line 565, in makeNeighborFigure
    cmd = self.__pathDot + " -T%s %s > %s" % (figFormat, dotfn, svgfn)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```
self.__pathDot was null without the GRAPHVIZ_DOT_BINARY environment variable.

Reply:
Weird, I wonder why this wasn't erroring in Azure before then, but unfortunately we've lost that history now to check. Thanks for catching and fixing that though. Other changes LGTM!